src/pubtools/_marketplacesvm/cloud_providers/aws.py

```diff
@@ -613,6 +613,9 @@ def _post_publish(
         Returns:
             Tuple of PushItem and Publish results.
         """
+        # Check if this product is locked currently and wait for it to become unlocked
+        LOG.info("Checking for active changesets in: %s", push_item.dest[0])
+        self.publish_svc.wait_active_changesets(push_item.dest[0])
```

Comment:
It's possible we needn't  restrict anything. In such case; waiting would not be needed.
Can we move that  after `if restrict_version:`  ?
https://github.com/release-engineering/pubtools-marketplacesvm/pull/84/files#diff-e6a1ee7ea989034e1a8b7568e235d7623bdb0b1364b11b8f5853958e3957f61aR631

Reply:
Oh right, can do :)